search/core/src/main/java/io/em2m/search/core/daos/QueryTransformingSearchDao.kt

```diff
@@ -10,6 +10,8 @@ class QueryTransformingSearchDao<T>(
         val namedAggs: Map<String, Agg> = emptyMap(),
         delegate: SearchDao<T>) : SearchDaoWrapper<T>(delegate) {
 
+    var timeZone : String = "America/Los_Angeles"
```

Comment:
This is a major concurrency issue.  This field gets modified for requests that include a timeZone and will affect subsequent requests. My guess is that you don't have access to the request-level timeZone field.

One approach would be to implement a timeZone transformer that adds the timeZone from the request to the queries that understand timeZone and don't specify it.